include/http_handlers.h

```diff
@@ -270,6 +270,12 @@ class ImpuRegDataTask : public HssCacheTask
 
   ImplicitRegistrationSet* _irs = NULL;
 
+  // The last registration state retrieved from the cache for this user.
+  // -- UNKNOWN if we haven't queried the cache
+  // -- UNREGISTERED if we have queried the cache and there was no record
```

Comment:
Should be `NOT_REGISTERED` if we've queried the cache and it wasn't found.

Reply:
Given https://github.com/Metaswitch/homestead/pull/485/files#diff-2212aa57f8d205d68d8b41fd10193c82L836 I think this is what will happen as well.